plugins/modules/provision_workflow_manager.py

```diff
@@ -773,17 +742,42 @@ def get_wireless_params(self):
                 "managedAPLocations": self.validated_config.get("managed_ap_locations"),
             }
         ]
-
-        if not (wireless_params[0].get("managedAPLocations") and isinstance(wireless_params[0].get("managedAPLocations"), list)):
-            msg = "Missing Managed AP Locations: Please specify the intended location(s) for the wireless device \
-                within the site hierarchy."
-            self.log(msg, "CRITICAL")
-            self.module.fail_json(msg=msg, response=[])
-
-        for ap_loc in self.validated_config.get("managed_ap_locations"):
-            if self.get_site_type(site_name_hierarchy=ap_loc) != "floor":
-                self.log("Managed AP Location must be a floor", "CRITICAL")
-                self.module.fail_json(msg="Managed AP Location must be a floor", response=[])
+        if self.get_ccc_version_as_integer() <= self.get_ccc_version_as_int_from_str("2.3.5.3"):
```

Comment:
Do this one after Ajith's commit.. Check with Ajith..

Reply:
Updated the code accordingly